crypt-md5.c

```diff
@@ -79,7 +80,7 @@ crypt_md5_rn (const char *phrase, const char *setting,
   const char *salt = setting;
 
   size_t salt_len;
-  size_t phrase_len;
+  size_t phrase_len = phr_size;
```

Comment:
As `phrase_len` is used as a constant during the code, it should be replaced by directly using the passed parameter only.  The same goes for `salt_len` in the line above.

Reply:
I will replace phrase_len with phr_size (here and elsewhere).  salt_len, however, is not the same as set_size; if the setting string is a previously generated hash, it will be shorter.